docs/design/distributed-framework-plugins.md

```diff
@@ -199,6 +193,108 @@ func (tp *tensorflowPlugin) getClusterInfo(job *batch.Job) clusterInfo {
 }
 ```
 
+#### Pytorch Plugin
+
+Similar to the tensorflow plugin, firstly we must know the cluster role of task in volcano job, and the port to be exposed. And this information can be passed by plugin arguments, which is defined in job spec.
+
+```yaml
+spec:
+  plugins:
+    # set pytorch plugin
+    pytorch: ["--master=master","--worker=worker","--port=23456"]
+```
+
+In the implementation of `pytorchPlugin`, these arguments will be parsed.
+
+```go
+// pytorchPlugin is plugin for pytorch framework
+type pytorchPlugin struct {
+	pytorchArguments []string
+	clientset        pluginsinterface.PluginClientset
+	masterName       string
+	workerName       string
+	port             int
+}
+// parse all arguments
```

Comment:
Perhaps the statement for API design or struct is enough. Complete implementation or functions are not required. Let's keep the desgin doc brief and to the point.